requirements.txt

```diff
@@ -15,3 +15,4 @@ qds_sdk>=1.10.1
 requests>=2.20
 retrying>=1.3.3
 urllib3>=1.24.2,<1.25
+py2neo
```

Comment:
Shouldn't we specify version of py2neo that is used and tested?

Reply:
Added :)